schema/archive.json

```diff
@@ -0,0 +1,210 @@
+                  "title": "Build type",
+                  "description": "Build type",
+                  "enum": [
+                    "nightly",
+                    "beta",
+                    "aurora",
+                    "release",
+                    "esr"
+                  ]
+                }
+              }
+            },
+            "target": {
+              "type": "object",
+              "additionalProperties": false,
+              "required": [
+                "product",
+                "version_string",
+                "major_version",
+                "release_version",
+                "version_sort"
+              ],
+              "properties": {
+                "product": {
+                  "type": "string",
+                  "title": "Product name",
+                  "description": "Name of the product the build is about"
+                },
+                "version_string": {
+                  "type": "string",
+                  "title": "Version",
+                  "description": "Version string. (i.e: 52.1.1 or 54.0b7)"
+                },
+                "major_version": {
+                  "type": "string",
+                  "title": "Major version",
+                  "description": "Major version. (i.e: 52.1 or 54.0)"
+                },
+                "release_version": {
+                  "type": "string",
+                  "title": "Release version",
+                  "description": "Release version. (i.e: 52.1.1 or 54.0)"
+                },
+                "beta_number": {
+                  "type": "integer",
+                  "title": "Beta number",
+                  "description": "Beta number (i.e: 7)"
+                },
+                "version_sort": {
+                  "type": "string",
+                  "title": "A value to sort records by version number",
+                  "description": "A value to sort records by version number. (i.e: 052001001r000 (release), 054000000a002 (nightly), 054000000b007 (beta), 045009000x000 (esr))"
+                },
+                "featured_version": {
+                  "type": "boolean",
+                  "title": "Featured version?"
+                },
+                "has_builds": {
+                  "type": "boolean",
+                  "title": "Has builds?"
+                },
+                "is_rapid_beta": {
+                  "type": "boolean",
+                  "title": "Is rapid beta?"
+                }
+              }
+            },
+            "source": {
+              "type": "object",
+              "additionalProperties": false,
+              "properties": {
+                "tree": {
+                  "type": "string",
+                  "title": "Tree",
+                  "description": "i.e mozilla-central"
+                },
+                "revision": {
+                  "type": "string",
+                  "title": "Revision number in the tree"
+                }
```

Comment:
I just realized that I didn't understand why `product` is part of `source`, and I was going to ask here, but I see that it isn't here, I guess also because this schema is aspirational?